src/components/measureLanding/measureList/MeasureList.tsx

```diff
@@ -79,6 +79,7 @@ export default function MeasureList(props: {
   measureList: Measure[];
   selectedIds: object;
   changeSelectedIds;
+  setSelectedIds: Function;
```

Comment:
Is setSelectedIds similar to changeSelectedIds? changeSelectedIds seems to do: setSelectedIds

Reply:
changeSelectIds are for single ones, but in my case I'll be changing the Ids for multiple ones

Reply:
@sb-prateekkeerthi Thanks for explanation!